.github/workflows/build-vsix.yml

```diff
@@ -22,11 +22,16 @@ jobs:
     steps:
     # Init
     - name: Checkout
-      uses: actions/checkout@v2.3.4
+      uses: actions/checkout@v4.0.0
+      with:
+        fetch-depth: 1
+        show-progress: false
     - name: Setup MSBuild.exe (add msbuild to PATH)
-      uses: microsoft/setup-msbuild@v1.0.3
+      uses: microsoft/setup-msbuild@v1.3.1
       with:
-        vs-prerelease: true
+        vs-version: '[17.0,18.0)'
```

Comment:
The _Build_ step is failing due to warnings as errors, with a false positive on [CA1825](https://learn.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1825).

The false positive is reported on usages of the attributes
- [DiagnosticAnalyzerAttribute](https://learn.microsoft.com/dotnet/api/microsoft.codeanalysis.diagnostics.diagnosticanalyzerattribute.-ctor)
- [ExportCodeFixProviderAttribute](https://learn.microsoft.com/dotnet/api/microsoft.codeanalysis.codefixes.exportcodefixproviderattribute.-ctor)
- [ExportCodeRefactoringProviderAttribute](https://learn.microsoft.com/dotnet/api/microsoft.codeanalysis.coderefactorings.exportcoderefactoringproviderattribute.-ctor)

invoked like

```CSharp
[DiagnosticAnalyzer(LanguageNames.CSharp)]
[ExportCodeFixProvider(LanguageNames.CSharp, Name = nameof(MyCodeFixProvider))]
[ExportCodeRefactoringProvider(LanguageNames.CSharp, Name = nameof(MyCodeRefactoringProvider))]
```

where we explicitly pass the argument `LanguageNames.CSharp` to the first parameter `string firstLanguage` of the attribute constructor, but we implicitly pass the argument `call !!0[] [System.Runtime]System.Array::Empty<string>()` to the second "params" parameter `params string[] additionalLanguages` of the attribute constructor.

I hope to be able to just "merge it off" via #106.

Reply:
Was fixed with the merge indeed.